crates/iota-sdk-ffi/src/types/struct_tag.rs

```diff
@@ -39,21 +42,18 @@ impl From<iota_types::TypeParseError> for TypeParseError {
 ///
 /// UNDERSCORE = %x95
 /// ```
-#[derive(Clone, Debug, derive_more::From, uniffi::Object)]
-pub struct Identifier(iota_types::Identifier);
+#[derive(Clone, Debug, Ord, PartialOrd, Eq, PartialEq, Hash, derive_more::From, uniffi::Object)]
+pub struct Identifier(pub iota_types::Identifier);
 
+#[uniffi::export]
 impl Identifier {
     #[uniffi::constructor]
-    pub fn new(identifier: impl AsRef<str>) -> Result<Self, TypeParseError> {
```

Comment:
Is the FFI version of `TypeParseError` even needed anymore then?

Reply:
no, probably not

Reply:
removed it